kani-driver/src/args/mod.rs

```diff
@@ -60,14 +61,29 @@ const DEFAULT_OBJECT_BITS: u32 = 16;
     version,
     name = "kani",
     about = "Verify a single Rust crate. For more information, see https://github.com/model-checking/kani",
-    args_override_self = true
+    args_override_self = true,
+    subcommand_negates_reqs = true,
+    subcommand_precedence_over_arg = true,
+    args_conflicts_with_subcommands = true
 )]
 pub struct StandaloneArgs {
     /// Rust file to verify
-    pub input: PathBuf,
+    #[arg(required = true)]
+    pub input: Option<PathBuf>,
 
     #[command(flatten)]
     pub verify_opts: VerificationArgs,
+
+    #[command(subcommand)]
+    pub command: Option<StandaloneSubcommand>,
+}
+
+/// Kani takes optional subcommands to request specialized behavior.
+/// When no subcommand is provided, there is an implied verification subcommand.
+#[derive(Debug, clap::Subcommand)]
+pub enum StandaloneSubcommand {
```

Comment:
Does `assess` not work in stand-alone mode?

Reply:
Nope. https://github.com/model-checking/kani/issues/1798